src/oop/object_interactions/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,50 @@
+# oop-object_interactions-medium-q2
+
+- Java question
+
+## Question Text
+
+Many people keep secrets that they only share with a trusted friend. Make a class called **Person** that has a secret 
```

Comment:
Please add that they also need a second class; otherwise they might miss it at the bottom.